setup.py

```diff
@@ -232,6 +232,22 @@ def configure_server_mode(configure, settings):
     for include in includes:
         settings.include(include)
 
+def configure_libreoffice(configure, settings):
+    print(INFO + textwrap.dedent(u"""
+            Application uses libreoffice package. If you don't have this package installed, 
+            it can be used for mock its function.""") + RESET)
```

Comment:
Nejak tu vetu neviem sparsovat. Nevypadli Ti tam nejake slova? Skus to preformulovat. Rovnako pri imagemagicu.

Reply:
:ok;